### PR TITLE
chore: fix pandas flaky test

### PR DESCRIPTION
**Pull request by RomanBredehoft**
this small flaky test made the weekly and other PRs) sometime fail : https://github.com/zama-ai/concrete-ml/actions/runs/9414309379/job/25933015157

### CODE REVIEW

File: tests/pandas/test_pandas.py [hunk 1]
```diff
@@ -714,7 +714,7 @@ def check_invalid_schema_values():
     ):
         client.encrypt_from_pandas(pandas_df, schema=schema_float_column)
 
-    schema_float_oob = {f"{feat_name}_float_1": {"min": float_min // 2, "max": float_max // 2}}
```
Comment (fd0r) [inline tests/pandas/test_pandas.py:717]: I'm fine with that but could you maybe explain a little more what's happening here? Having min=max seems kinda weird.

**Review by fd0r — DISMISSED** [review]
Some comment but fine with me

File: tests/pandas/test_pandas.py (hunk 1 above)
Comment (RomanBredehoft) [reply tests/pandas/test_pandas.py:717]: we want to always trigger then catch e error, so obviously we need weird configs like this 😅

Comment (RomanBredehoft) [reply tests/pandas/test_pandas.py:717]: added a comment

**Review by fd0r — APPROVED** [review]
Perfect thanks!

**Pull request merged**